### ISSUE TITLE
Implement Hyperparameter tuning

### ISSUE THREAD

**Issue by Pritamstudent** [Rakesh9100/ML-Project-Drug-Review-Dataset#148]
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

Do hyperparameter tuning of XGBoost and Adaboost model to find the best parameters and find highest accuracy  score. 

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
